pkg/storage/node_liveness.go

```diff
@@ -151,6 +153,13 @@ func NewNodeLiveness(
 
 var errNodeDrainingSet = errors.New("node is already draining")
 
+func (nl *NodeLiveness) sem(nodeID roachpb.NodeID) chan struct{} {
+	if nodeID == nl.gossip.NodeID.Get() {
```

Comment:
It's mildly unfortunate that 2/3 of the callers to this method also have a static call to `nl.gossip.NodeID.Get()` and thus know they always want `selfSem`.